ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -0,0 +1,606 @@
+
+// TODO: Classic buff icon by level
+export const AllStatsBuff = withLabel(
+	makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(9885), impId: ActionId.fromSpellId(17055), fieldName: 'giftOfTheWild'}),
+	'Mark of the Wild',
+);
+
+// Separate Strength buffs allow us to use a boolean pickers for Horde specifically
+export const AllStatsPercentBuffAlliance = InputHelpers.makeMultiIconInput([
+	makeBooleanIndividualBuffInput({actionId: ActionId.fromSpellId(20217), fieldName: 'blessingOfKings', faction: Faction.Alliance}),
+	makeBooleanRaidBuffInput({actionId: ActionId.fromSpellId(409580), fieldName: 'aspectOfTheLion', faction: Faction.Alliance}),
+], 'Stats %');
+
+export const AllStatsPercentBuffHorde = withLabel(
+	makeBooleanRaidBuffInput({actionId: ActionId.fromSpellId(409580), fieldName: 'aspectOfTheLion', faction: Faction.Horde}),
+	'Stats %',
+);
+
+// TODO: Classic armor buff ranks
+export const ArmorBuff = InputHelpers.makeMultiIconInput([
+	makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(10293), impId: ActionId.fromSpellId(20142), fieldName: 'devotionAura'}),
+	makeBooleanRaidBuffInput({actionId: ActionId.fromItemId(1478), fieldName: 'scrollOfProtection'}),
+], 'Armor');
+
+export const StaminaBuff = InputHelpers.makeMultiIconInput([
+	makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(10938), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude'}),
+	// makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(10937), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 48, maxLevel: 59}),
+	// makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(2791), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 36, maxLevel: 47}),
+	// makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(1245), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 24, maxLevel: 35}),
+	// makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(1244), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 12, maxLevel: 23}),
+	// makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(1243), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 1, maxLevel: 11}),
+	makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(11767), impId: ActionId.fromSpellId(18696), fieldName: 'bloodPact'}),
+	makeBooleanRaidBuffInput({actionId: ActionId.fromItemId(10307), fieldName: 'scrollOfStamina'}),
+], 'Stamina');
+
+// Separate Strength buffs allow us to use boolean pickers for each
+export const StrengthBuffAlliance = InputHelpers.makeMultiIconInput([
+	makeTristateIndividualBuffInput({actionId: ActionId.fromSpellId(25291), impId: ActionId.fromSpellId(20048), fieldName: 'blessingOfMight', faction: Faction.Alliance}),
+	makeBooleanRaidBuffInput({actionId: ActionId.fromSpellId(425600), fieldName: 'hornOfLordaeron', faction: Faction.Alliance}),
+], 'Strength / Agility');
```

Comment:
After thinking about this more, I think we should name this `PaladinPhysicalBuff` for maximum clarity, and then make the label `Paladin Physical`, which isn't more characters than `Strength / Agility` and is more clear.